include/oneapi/dpl/pstl/hetero/dpcpp/sycl_defs.h

```diff
@@ -207,6 +207,18 @@ constexpr __target __target_device =
     __target::global_buffer;
 #endif
 
+template <typename DataT>
+using __buffer_allocator =
+#if __LIBSYCL_VERSION >= 50700
```

Comment:
it looks that we have not an alternate definition in case of __LIBSYCL_VERSION < 50700

Reply:
fixed